components/brave_rewards/browser/database/publisher_info_database.h

```diff
@@ -200,6 +200,7 @@ class PublisherInfoDatabase {
   std::unique_ptr<DatabaseUnblindedToken> unblinded_token_;
   std::unique_ptr<DatabaseContributionInfo> contribution_info_;
   std::unique_ptr<DatabasePendingContribution> pending_contribution_;
+  std::unique_ptr<DatabaseMediaPublisherInfo> media_publisher_info;
```

Comment:
member variable should end with an underscore

Reply:
done